featuretools/primitives/utils.py

```diff
@@ -39,14 +39,14 @@ def get_transform_primitives():
 
```

Comment:
Should there be a comment above to say that `get_aggregation_primitives` will now return all primitives? Whether or not they are compatible with dask/koalas? I know some users take `get_aggregation_primitives` and put it directly into dfs...